bench/wiredtiger-copy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# 2021-08-11 J. Dagger
+#
+# Copy built wiredtiger executables over to QEMU guest
```

Comment:
Looks like this script assumes the guest is running? If that's true, please add the assumption in the comment. What happens if the guest is not running?

Reply:
I'll add that assumption. Right now, it will build stuff but then hit an error with ssh. I'll add a check to print an error and return an exit code if it doesn't work

Reply:
That assumption in the file comments